roles/pulp_common/README.md

```diff
@@ -134,7 +134,21 @@ RPM packages instead if this variable is set. Other distro packaging formats may
   (RPM) "packages".
   Defaults to "pip".
 
-If it is set to "packages", the following variables are used, or behave *differently* from above:
+If it is set to "packages", the installer is in packges mode, which has the following **limitations**:
+
+* Only CentOS/RHEL 7 and CentOS/RHEL 8 are supported.
+* Not all plugins are available from the default repo. To determine which plugins are available,
+  follow [this link](https://yum.theforeman.org/pulpcore/), browse to the repo for your
+  pulp_version and distribution, and search for "python3-pulp-".
+* The default repo (from yum.theforeman.org, see `pulp_pkg_repo`) is not tested for every possible
+  pulpcore usage, and is thus not officially supported by the Foreman project.
+* The default repo may be for an older minor branch of pulpcore. e.g., if pulp_installer is 3.9.y,
+  it may install 3.8 instead.
+* The default repo may be changed to the current minor branch during any micro release. E.g.,
+  pulp_installer 3.9.0 may install/upgrade to Pulp 3.8, while pulp_installer 3.9.1 may
+  install/upgrade to Pulp 3.9.
```

Comment:
I don't understand this section.

Don't you set `pulp_pkg_repo` to `https://yum.theforeman.org/pulpcore/X.Y`?

On the other hand, we don't build every Y-stream of yours.

Reply:
what I am trying to say is that unlike with a regular `pip` mode install, where pulp_installer 3.9 would always install pulp 3.9.z by default, `packages` mode may install an older 3.y version of Pulp by default.

Where "by default" means that pulp_pkg_repo is defaulted.

Reply:
I clarified this in the docs.